client/connection-string/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I don't understand why this is in another package. Are we expecting something to use this outside of client?

Reply:
It was pulled almost verbatim from the PG package back when I was trying to make the two mirror images of each other. PG had it structured as a separate npm package, which is why I kept it.

At this point it doesn't matter. My latest push combines it into the main package.